swift/Samples/SplitScreenContainer/Sources/SplitScreenContainerViewController.swift

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Workflow
+import WorkflowUI
+
+extension ViewRegistry {
+    
+    public mutating func registerSplitScreenContainer() {
+        self.register(screenViewControllerType: SplitScreenContainerViewController.self)
+    }
+    
+}
+
+
+final class SplitScreenContainerViewController: ScreenViewController<SplitScreenContainerScreen> {
+
+    var leftContentViewController: ScreenViewController<AnyScreen>? = nil
+    var leftContainerView: UIView
+
+    var separatorView: UIView
+
+    var rightContentViewController: ScreenViewController<AnyScreen>? = nil
+    var rightContainerView: UIView
+
+    required init(screen: SplitScreenContainerScreen, viewRegistry: ViewRegistry) {
+        self.leftContainerView = UIView()
+        self.leftContainerView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Move to `viewDidLoad`

Reply:
Resolved via https://github.com/square/workflow/pull/925/commits/205354d6c1a472cda836621d53138f15ad8f3959